### PR TITLE
[docs]: Fix documentation for Iframe Component 

### PR DESCRIPTION
**Pull request by dagenius007**
PR Description
This PR fixes the issue: https://github.com/ToolJet/ToolJet/issues/10971

## Summary of Changes:
Replaced "Fx" with "fx" (lowercase and bold) across the document.
Removed 24px padding-bottom from h2 headers, retaining only 24px padding-top.
Updated the word "widget" to "component" throughout the documentation.
Added a divider before the "Styles" section.

## Affected Files:
ToolJet/docs/docs/widgets/iframe.md
ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/iframe.md

### CODE REVIEW

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @dagenius007, you were supposed to make changes in LTS version i.e. 2.50.0 but I think you have made changes in 2.5.0
Can you please revert all the changes from 2.5.0 and replicate them in 2.50.0?

File: docs/docs/widgets/iframe.md [hunk 1]
```diff
@@ -2,21 +2,22 @@
...
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> |
+| :--------------------------------------------- | :------------------------------------------------ |
+| URL                                            | Set the **URL** of the page to embed.             |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Component Specific Actions (CSA)
```
Comment (RudraPratik30) [inline docs/docs/widgets/iframe.md:23]: Hey @dagenius007, I think here on line no. 28 you forgot to remove paddingBottom, please make the change.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

File: docs/docs/widgets/iframe.md (hunk 1 above)
Comment (dagenius007) [reply docs/docs/widgets/iframe.md:23]: @RudraPratik30 resolved

**Review by RudraPratik30 — APPROVED** [review]

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Review by johnsoncherian — APPROVED** [review]

**Pull request merged**